crates/ef-testing/src/evm_sequencer/constants.rs

```diff
@@ -72,12 +72,14 @@ lazy_static! {
     pub static ref CONTRACT_ACCOUNT_CLASS: LegacyContractClass = serde_json::from_reader::<_, LegacyContractClass>(std::fs::File::open("../../lib/kakarot/build/contract_account.json").unwrap()).unwrap();
     pub static ref EOA_CLASS: LegacyContractClass = serde_json::from_reader::<_, LegacyContractClass>(std::fs::File::open("../../lib/kakarot/build/externally_owned_account.json").unwrap()).unwrap();
     pub static ref PROXY_CLASS: LegacyContractClass = serde_json::from_reader::<_, LegacyContractClass>(std::fs::File::open("../../lib/kakarot/build/proxy.json").unwrap()).unwrap();
+    pub static ref FEE_TOKEN_CLASS: LegacyContractClass = serde_json::from_reader::<_, LegacyContractClass>(std::fs::File::open("../../lib/kakarot/build/fixtures/ERC20.json").unwrap()).unwrap();
```

Comment:
I think we can even do `include_str!` macro, to inline the string at compile time
and use serde_json::from_str 


that is in case the .json isn't too big, we might win a bit of speed;D

Reply:
Otherwise, might be useful to abstract the logic of these LegacyContractClass? such as 
```rust
fn load_legacy_contract_class(path: &str) -> Result<LegacyContractClass, Box<dyn std::error::Error>> {
    let file = std::fs::File::open(path)?;
    let class = serde_json::from_reader::<_, LegacyContractClass>(file)?;
    Ok(class)
}

pub static ref EOA_CLASS: LegacyContractClass = load_legacy_contract_class("../../lib/kakarot/build/externally_owned_account.json").expect("Failed to load EOA_CLASS");
pub static ref PROXY_CLASS: LegacyContractClass = load_legacy_contract_class("../../lib/kakarot/build/proxy.json").expect("Failed to load PROXY_CLASS");
pub static ref FEE_TOKEN_CLASS: LegacyContractClass = load_legacy_contract_class("../../lib/kakarot/build/fixtures/ERC20.json").expect("Failed to load FEE_TOKEN_CLASS");

```

Reply:
for include_str!, you mean for all the contract classes? That might be a little big no?

Reply:
maybe a bit too big yeah, you're right

Reply:
but i don't know how/where to find the optimum between binary size vs. runtime overhead of I/O

Reply:
I'm not sure it changes much in this case, if you add `include_str`, binary will contain all this data and load it quickly so `initialize` of the sequencer might win some time (on the first init) but pretty sure that's it